### DISCUSSION TITLE
server crash when connect esp8266 to socket

### DISCUSSION THREAD

**Discussion by majidkazemi2000** [Links2004/arduinoWebSockets#692] [General]
hi , when i connect esp8266 to socket.io and nodejs server , the server crashes and gives this error :
{ errno: -4077, code: 'ECONNRESET', syscall: 'read' } in socket.io

**Comment by WellingtonSouzaAbreu** [comment]
Olá
Já resolveu?
Se não, me mande os códigos do esp8266, do server, por favor
